types/swiper/index.d.ts

```diff
@@ -1,461 +1,1621 @@
 // Type definitions for Swiper 4.2
 // Project: https://github.com/nolimits4web/Swiper
-// Definitions by: Sebastián Galiano <https://github.com/sgaliano>, Luca Trazzi <https://github.com/lucax88x>, Eugene Matseruk <https://github.com/ematseruk>
+// Definitions by: Sebastián Galiano <https://github.com/sgaliano>, Luca Trazzi <https://github.com/lucax88x>, Eugene Matseruk <https://github.com/ematseruk>, Luiz M. <https://github.com/odahcam>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.7
-
-type CommonEvent =
-    | 'init'
-    | 'beforeDestroy'
-    | 'slideChange'
-    | 'slideChangeTransitionStart'
-    | 'slideChangeTransitionEnd'
-    | 'slideNextTransitionStart'
-    | 'slideNextTransitionEnd'
-    | 'slidePrevTransitionStart'
-    | 'slidePrevTransitionEnd'
-    | 'transitionStart'
-    | 'transitionEnd'
-    | 'touchStart'
-    | 'touchMove'
-    | 'touchMoveOpposite'
-    | 'sliderMove'
-    | 'touchEnd'
-    | 'click'
-    | 'tap'
-    | 'doubleTap'
-    | 'imagesReady'
-    | 'progress'
-    | 'reachBeginning'
-    | 'reachEnd'
-    | 'fromEdge'
-    | 'setTranslate'
-    | 'setTransition'
-    | 'resize';
-
-type PaginationEvent = 'paginationRender' | 'paginationUpdate';
-type AutoplayEvent = 'autoplayStart' | 'autoplayStop' | 'autoplay';
-type LazyLoadingEvent = 'lazyImageLoad' | 'lazyImageReady';
-
-type SwiperEvent = CommonEvent | PaginationEvent | AutoplayEvent | LazyLoadingEvent;
-
-interface NavigationOptions {
-    nextEl?: string | HTMLElement;
-    prevEl?: string | HTMLElement;
-    hideOnClick?: boolean;
-    disabledClass?: string;
-    hiddenClass?: string;
-}
+// TypeScript Version: 2.9
+// declare namespace swiper {
```

Comment:
I think this is redundant line

Reply:
You're right, I just left it there because someone on Gitter said me that I shouldn't declare my typings using module, but because of Swiper's use of that custom ism